src/main/java/com/cburch/logisim/fpga/hdlgenerator/LedArrayGenericHDLGeneratorFactory.java

```diff
@@ -319,27 +319,27 @@ public static boolean requiresClock(char typeId) {
               + HDL.BracketOpen()
               + pin
               + HDL.BracketClose(),
-              rOn,
-              rOff,
-              pinName));
+                                      rOn,
+                                      rOff,
+                                      pinName));
```

Comment:
Just as side note, but again this space thingy....

Reply:
Fixed.